05_stringer/alextmz/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+type iPAddr [4]byte
```

Comment:
This really should be called `ipAddr` - when you have an initialism/acronym at the start of a symbol that is private, you should make it all lowercase - `ip` in this case. If it were public, it would be `IP`. See https://github.com/golang/go/wiki/CodeReviewComments#initialisms.

Reply:
Agreed, I've been paying attention to these but started on the later labs.